osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -0,0 +1,256 @@
+import os
+import shutil
+import subprocess
+import tempfile
+import time
+import sys
+import traceback
+
+from celery import shared_task
+
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+from django.core.files import File
+from django.core.files.storage import FileSystemStorage
+from django.contrib import messages
+
+from excerptconverter.baseexcerptconverter import BaseExcerptConverter
+from excerptconverter import ConverterHelper
+
+from osmaxx.excerptexport import models
+
+
+private_storage = FileSystemStorage(location=settings.PRIVATE_MEDIA_ROOT)
```

Comment:
centralize creation of `private_storage` (also used in `views.py` and in dummy converter)